metron-deployment/packaging/docker/rpm-docker/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+rm -rf SRPMS/ RPMS/ && \
+rpmbuild -v -ba --define "_topdir $(pwd)" --define "_version 0.2.0BETA" SPECS/metron.spec && \
+rpmlint -i SPECS/metron.spec RPMS/*/metron* SRPMS/metron
```

Comment:
rpmlint checks the spec files as well as the RPM's themselves - I'd recommend keeping it for verification that we:
1. follow RPM standards
2. don't have errors (RPM can build with error code 0, but have actual issues for installation)